suite-native/atoms/src/Button/IconButton.tsx

```diff
@@ -1,93 +1,98 @@
-import React from 'react';
-import { TouchableOpacity } from 'react-native';
+import React, { useState } from 'react';
+import { Pressable, PressableProps } from 'react-native';
 
 import { Icon, IconName } from '@trezor/icons';
-import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import {
+    NativeStyleObject,
+    prepareNativeStyle,
+    mergeNativeStyles,
+    useNativeStyles,
+} from '@trezor/styles';
 
-import { ButtonColorScheme, ButtonSize } from './Button';
+import {
+    ButtonColorSchemeName,
+    ButtonSize,
+    buttonSchemeToColorsMap,
+    buttonStyle,
+    ButtonStyleProps,
+} from './Button';
+import { Box } from '../Box';
+import { Text } from '../Text';
 
-type IconButtonProps = {
+type IconButtonProps = Omit<PressableProps, 'style'> & {
     iconName: IconName;
-    onPress: () => void;
-    isRounded?: boolean;
-    colorScheme?: ButtonColorScheme;
+    colorScheme?: ButtonColorSchemeName;
```

Comment:
nit: I would drop the name or rename the property to colorSchemeName as well (super unimportant, just semantics)

Reply:
fixed in 0f7794d